mne_bids/inspect.py

```diff
@@ -39,6 +42,9 @@ def inspect_dataset(bids_path, l_freq=None, h_freq=None,
         To read a specific file, set all the :class:`mne_bids.BIDSPath`
         attributes required to uniquely identify the file: If this ``BIDSPath``
         is accepted by :func:`mne_bids.read_raw_bids`, it will work here.
+    find_flat : bool
+        Whether to auto-detect channels producing "flat" signals, i.e., with
+        unusually low variability.
```

Comment:
Since we're hard-setting the find_flat parameters, perhaps we can also repeat documentation here of how we find_flat?

E.g. find_flat looks for flat_segments of at least 50 ms and calls a channel flat if more then 5% of the channel is flat.

Reply:
I've added a note in d00c8f1